.env.example

```
@@ -25,6 +25,12 @@ RABBITMQ_TLS=false
 RABBITMQ_EXCHANGE=contact_messages
 RABBITMQ_QUEUE=contact_messages
 RABBITMQ_RETRY_DELAYS=1m,5m,30m,2h,12h
+RABBITMQ_HEARTBEAT=10s
+RABBITMQ_PUBLISHER_CONFIRMS=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_PUBLISHER_CONFIRMS key should go before the RABBITMQ_QUEUE key

(UnorderedKey)